docs/README.md

```diff
@@ -1,97 +1,62 @@
 # Minimum Boilerplate Theme
-Use this template to start a new Store Framework Theme from zero.
 
+The minimum Boilerplate Theme is basic store front model based on the VTEX IO Store Framework.
 
-## Table of Contents
-- [Getting Started](#getting-started)
-  * [Step 1. Complete the basic setup](#step-1-complete-the-basic-setup)
-  * [Step 3. Create a workspace](#step-3-create-a-workspace)
-  * [Step 3. Clone this resitory](#step-3-clone-this-resitory)
-  * [Step 4. Prepare your Manifest.json](#step-4-prepare-your-manifestjson)
-  * [Step 5. Install required apps](#step-5-install-required-apps)
-  * [Step 6. Uninstall any installed theme](#step-6-uninstall-any-installed-theme)
-  * [Step 7. Run and preview your store](#step-7-run-and-preview-your-store)
-- [Tips](#tips)
+It should be used only when you want to start a new store theme without any pre-set configurations, as is the case with [Store Theme](https://github.com/vtex-apps/store-theme). 
 
-## Getting Started
+While Store Theme gives developers a ready-to-go default store front structure, the Minimum Boilerplate Theme will enable you to build you store freely from scratch.
 
-### Step 1. Complete the basic setup
-Go to the [Basic Setup Guide](https://vtex.io/docs/getting-started/build-stores-with-vtex-io/2) in the VTEX IO docs and complete it.
+## Configuration
 
-You should have the VTEX command line interface installed and
-where to go after you finish.
+### Step 1 -  Basic setup
 
-### Step 3. Create a workspace
-It is a good practice to always begin a development session in an environment
-separated from production. We call then workspace.
+Access the VTEX IO [basic setup guide](https://vtex.io/docs/getting-started/build-stores-with-vtex-io/2) and follow all the given steps. 
 
-Create a workspace with the following command:
+By the end of the setup, you should have the VTEX command line interface (Toolbelt) installed along with a developer workspace you can work in.
 
-```bash
-vtex use <YOUR_NAME>
-```
+### Step 2 - Cloning the Minimum Boilerplate Theme repository
+
+[Clone](https://help.github.com/en/github/creating-cloning-and-archiving-repositories/cloning-a-repository) this repository to your local files to be able to effectively start working on it.
+
+Then, access the repository's directory using your terminal. 
```

Comment:
Why did you skip the "create workspace" step? @guerreirobeatriz

Reply:
Because it is already included in the Basic Setup step